app/src/sharedTest/java/org/oppia/android/app/testing/InputInteractionViewTestActivityTest.kt

```diff
@@ -596,7 +596,7 @@ class InputInteractionViewTestActivityTest {
   }
 
   @Test
-  fun testNumericInputInteractionView_withInputtedLongNonDecimalNumber_clickSubmitButton_numberTooLongErrorIsDisplayed() { // ktlint-disable max-line-length
+  fun testNumericInputInteractionView_withLongNonDecimalNum_onSubmit_numTooLongErrorIsDisplayed() {
```

Comment:
testNumericInputInteractionView_withLongNumber_onSubmit_numberTooLongErrorIsDisplayed

we need consistency, as we are writing `number` everywhere.

Reply:
Done